testsuite/features/secondary/srv_monitoring.feature

```diff
@@ -26,8 +26,8 @@ Feature: Disable and re-enable monitoring of the server
     And I should see a list item with text "Tomcat (Java JMX)" and a failing bullet
     And I should see a "Restarting Tomcat and Taskomatic is needed for the configuration changes to take effect." text
     And file "/etc/rhn/rhn.conf" should contain "prometheus_monitoring_enabled = 0" on server
-    And file "/etc/sysconfig/tomcat" should not contain "Dcom.sun.management.jmxremote.port=3333 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Djava.rmi.server.hostname=" on server
-    And file "/etc/rhn/taskomatic.conf" should not contain "Dcom.sun.management.jmxremote.port=3334 -Dcom.sun.management.jmxremote.ssl=false -Dcom.sun.management.jmxremote.authenticate=false -Djava.rmi.server.hostname=" on server
+    And port "3333" should be closed
+    And port "3334" should be closed
     And file "/usr/lib/systemd/system/tomcat.service.d/jmx.conf" should not exist on server
     And file "/usr/lib/systemd/system/taskomatic.service.d/jmx.conf" should not exist on server
```

Comment:
Good to know. Thanks Johannes. I will add 2 additional checks for the ports then. I think it makes sense to include the changes here since it is related the the work is not that much.

Reply:
Added with 2affe6f1f2d77beaf2f21cbf442f7e67141de526.